tests/behat/behat.yml

```diff
@@ -8,6 +8,8 @@ default:
         - FeatureContext
         - ContentPoolContext
         - ParagraphContext
+        - FormContext
+        - MediaReplicationContext
```

Comment:
Let's avoid introducing further project-specific context when possible. When step definitions are generally useful, we should put them in behat-drupal-utils.